app/subscriber/src/features/search-page/SearchPage.tsx

```diff
@@ -40,47 +30,18 @@ export const SearchPage: React.FC<ISearchType> = ({ showAdvanced }) => {
     },
     { findContentWithElasticsearch, storeSearchFilter },
   ] = useContent();
-  const navigate = useNavigate();
-  const [{ frontPageImagesMediaTypeId, settings, isReady }] = useLookup();
+  const [{ frontPageImagesMediaTypeId }] = useLookup();
   const genQuery = useElastic();
   const [, { getFilter }] = useFilters();
   const [{ filter: activeFilter }, { storeFilter }] = useProfileStore();
 
-  const [content, setContent] = React.useState<IContentModel[]>([]);
-  const [activeContent, setActiveContent] = React.useState<IContentModel | null>(null);
-  const [playerOpen, setPlayerOpen] = React.useState<boolean>(false);
+  const [content, setContent] = React.useState<IContentSearchResult[]>([]);
   const [selected, setSelected] = React.useState<IContentModel[]>([]);
   const [isLoading, setIsLoading] = React.useState(false);
-  const [showPage, setShowPage] = React.useState<string[]>([]);
-  const [showNewWindow, setShowNewWindow] = React.useState<string[]>([]);
-  const [hideSource, setHideSource] = React.useState<string[]>([]);
   const [init, setInit] = React.useState(true); // React hooks are horrible...
 
   const filterId = id ? parseInt(id) : 0;
 
-  React.useEffect(() => {
-    if (isReady) {
-      const showPageIds = settings.find(
```

Comment:
Implemented new window, need to re implement show page, currently just shows page number if one exists